evm/src/core/utils.rs

```diff
@@ -7,6 +7,14 @@ use primitive_types::U256;
 pub const USIZE_MAX: U256 = U256([usize::MAX as u64, 0, 0, 0]);
 /// Precalculated `u64::MAX` for `U256`
 pub const U64_MAX: U256 = U256([u64::MAX, 0, 0, 0]);
+/// Precalculated `zero` value for `U256`
+pub const U256_ZERO: U256 = U256([0, 0, 0, 0]);
```

Comment:
Could use `U256::zero()` instead here if you wanted to since it is a `const` function. Similarly for `U256::one` below.

Reply:
Done: 2c00bbe6ae891b965ee62ffc28af846ac2e945d2